tests/test_code_review_fixes.py

```
@@ -286,9 +299,8 @@ def identity(x: int) -> int:
 
         with warnings.catch_warnings(record=True) as w:
             warnings.simplefilter("always")
-            result = runner.run(graph, {"x": 10}, select=["typo_result"])
+            result = runner.run(graph, {"x": 10}, select=["typo_result"], on_missing="warn")
```

Ruff (0.15.0)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)